### ISSUE TITLE
`svelte/block-lang` does not check `<script context="module">` when a component script tag is present

### ISSUE THREAD

**Issue by RubyTunaley** [sveltejs/eslint-plugin-svelte#507]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

v8.42.0

### What version of `eslint-plugin-svelte` are you using?

v2.30.0

### What did you do?

<details>
<summary>Configuration</summary>

```js
const options = {
	env: {
		browser: true
	},
	parser: '@typescript-eslint/parser',
	parserOptions: {
		extraFileExtensions: ['.svelte'],
		ecmaVersion: 'latest',
		sourceType: 'module',
		project: true,
		tsconfigRootDir: __dirname
	},
	overrides: [
		{
			files: ['*.svelte'],
			parser: 'svelte-eslint-parser',
			parserOptions: {
				parser: '@typescript-eslint/parser',
				extraFileExtensions: ['.svelte'],
				tsconfigRootDir: __dirname
			}
		}
	],
	plugins: ['@typescript-eslint', 'eslint-plugin-svelte'],
	root: true,
	rules: {
		'svelte/block-lang': [2, {
			script: ['ts']
		}]
	},
	reportUnusedDisableDirectives: true
};

module.exports = options;
```
</details>

```svelte
<script context="module"></script>
<script lang="ts"></script>
```

### What did you expect to happen?

Linting error: error  The lang attribute of the <script> block should be "ts"  svelte/block-lang

### What actually happened?

No linting error was raised.

### Link to **GitHub Repo** with Minimal Reproducible Example

https://github.com/RubyTunaley/svelte-eslint-block-lang-bug

**Comment by marekdedic** [comment]
Hmm,
I tried adding tests for this and can't replicate - am I doing something wrong?

https://github.com/marekdedic/eslint-plugin-svelte/tree/block-lang-context-module
